streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -215,12 +215,12 @@ public long size() {
     }
 
     /**
-     * Reject late events that arrive more than {@code afterWindowEnd}
+     * Reject out-of-order events that arrive more than {@code afterWindowEnd}
```

Comment:
Did we discuss defining "late" as out-of-order records that arrive after the window is closed? It seems like this would be the place to define the term "late" properly.

Reply:
I added a sentence to Windows JavaDoc -- do you think that is sufficient? Don't want to make the JavaDocs too complicated.